### ISSUE TITLE
Create standards for entering data

### ISSUE THREAD

**Issue by smitma09** [fitzco01/UltimateProject#1]
Will need to think about how data is entered once we get the new database up and running to create consistency from row to row within a table and allow for better sorting

i.e. Creating a dropdown to select "Division 1", "Division 3", or "Developmental" when add a team to make sure everything is formatted in a consistent way (Don't have to worry about people entering all sorts of variations ("d1", "D-I", "Division I", etc)

Talk to Matt or see https://github.com/fitzco01/UltimateProject/blob/master/serverFiles/finalDatabase.py for documentation on what we should be expecting for each field entry

**Comment by fitzco01** [comment]
Might be a good idea to follow this principle with other things too.  I'm thinking of region, state, conference on the team page, as well as position, height, and age on the player page.  We may need to talk about what possible options are for dropdowns that so that we cover our bases.

**Comment by smitma09** [comment]
Definitely, there are a ton of places where we'll need to control what gets entered somehow, competition division was just one area that came to mind

This is only semi-on topic, but as long as it's in my head it's worth noting that using single quotes or double quotes when entering data creates issues with the database (probably interprets them as indication of a string rather than straight text input). Just something to be aware of and address at some point

**Comment by fitzco01** [comment]
I'm not sure what you mean by that.  When I fill out a form I don't need to use quotes at all.  You must be talking about some part of php code?

**Comment by smitma09** [comment]
Right, you don't need to use quotes at all, but if you entered something like 

6'2"

for height when adding a player, that will cause issues when we try to read that entry from the database. That particular example is easy to avoid by adding 2 integer input fields (one for feet and one for inches) and then doing conversions behind the scenes, but if someone did use quotes when entering information for some reason, it'll likely create issues
